spec/features/admin/orders_spec.rb

```diff
@@ -196,20 +196,64 @@ def new_order_with_distribution(distributor, order_cycle)
       login_to_admin_as @enterprise_user
     end
 
-    context "viewing the edit page" do
-      it "shows the dropdown menu" do
+    feature "viewing the edit page" do
+      background do
+        Spree::Config[:enable_receipt_printing?] = true
```

Comment:
can we mock this? otherwise config will get cached and can affect other tests, especially in CI where we use parallel tests